<ui version="4.0" >
 <author>Detlev Offenbach &lt;detlev@die-offenbachs.de></author>
 <class>VariablesFilterDialog</class>
 <widget class="QDialog" name="VariablesFilterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Variables Type Filter</string>
  </property>
  <property name="whatsThis" >
   <string>&lt;b>Filter Dialog&lt;/b>
&lt;p> This dialog gives the user the possibility to select what kind of variables should &lt;b>not&lt;/b> be shown during a debugging session.&lt;/p></string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="2" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="localsLabel" >
     <property name="text" >
      <string>&amp;Locals Filter</string>
     </property>
     <property name="buddy" >
      <cstring>localsList</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="globalsLabel" >
     <property name="text" >
      <string>&amp;Globals Filter</string>
     </property>
     <property name="buddy" >
      <cstring>globalsList</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QListWidget" name="localsList" >
     <property name="toolTip" >
      <string>Locals Filter List</string>
     </property>
     <property name="whatsThis" >
      <string>&lt;b>Locals Filter List&lt;/b>
&lt;p>Select the variable types you want to be filtered out of the locals variables list.&lt;/p&lt;</string>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QListWidget" name="globalsList" >
     <property name="toolTip" >
      <string>Globals Filter List</string>
     </property>
     <property name="whatsThis" >
      <string>&lt;b>Globals Filter List&lt;/b>
&lt;p>Select the variable types you want to be filtered out of the globals variables list.&lt;/p&lt;</string>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>localsList</tabstop>
  <tabstop>globalsList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VariablesFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>14</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel" >
     <x>15</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VariablesFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel" >
     <x>84</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
